SwedbankPaySDK/Classes/Merchant Backend/Problems.swift

```diff
@@ -0,0 +1,184 @@
+//
+// Copyright 2020 Swedbank AB
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import Alamofire
+
+enum Problems {
+    static func parseProblem(
+        response: AFDataResponse<Data>
+    ) -> SwedbankPaySDK.Problem {
+        do {
+            return try parseProblemOrFail(response: response)
+        } catch {
+            return makeUnexpectedContentProblem(response: response)
+        }
+    }
+    
```

Comment:
Codacy found an issue: [Lines should not have trailing whitespace.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)